web-admin/src/features/public-urls/PublicURLsTable.svelte

```diff
@@ -0,0 +1,268 @@
+<script lang="ts">
+  import { writable } from "svelte/store";
+  import {
+    createSvelteTable,
+    flexRender,
+    getCoreRowModel,
+    getPaginationRowModel,
+    getSortedRowModel,
+  } from "@tanstack/svelte-table";
+  import type {
+    ColumnDef,
+    OnChangeFn,
+    SortingState,
+    TableOptions,
+  } from "@tanstack/svelte-table";
+  import PublicURLsActionsRow from "./PublicURLsActionsRow.svelte";
+  import DashboardLink from "./DashboardLink.svelte";
+  import ArrowDown from "@rilldata/web-common/components/icons/ArrowDown.svelte";
+  import type { V1MagicAuthToken } from "@rilldata/web-admin/client";
+
+  interface MagicAuthTokenProps extends V1MagicAuthToken {
+    dashboardTitle: string;
+  }
+
+  export let tableData: MagicAuthTokenProps[];
+  export let pageSize: number;
+  export let onDelete: (deletedTokenId: string) => void;
+  export let onNextPage: () => void;
+  export let hasNextPage: boolean;
+
+  let sorting: SortingState = [
+    {
+      id: "createdOn",
+      desc: true,
+    },
+  ];
+
+  function formatDate(value: string) {
+    return new Date(value).toLocaleDateString(undefined, {
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+      hour: "numeric",
+      minute: "numeric",
+    });
+  }
+
+  const columns: ColumnDef<MagicAuthTokenProps, any>[] = [
+    {
+      accessorKey: "name",
+      header: "Public URL name",
+      cell: ({ row }) =>
+        flexRender(DashboardLink, {
+          href: row.original.url,
+          title: row.original.name,
+        }),
+    },
+    {
+      accessorFn: (row) => row.dashboardTitle,
+      header: "Dashboard name",
```

Comment:
We should label this "Dashboard title" because "name" means "unique identifier" in Rill resources.